app/api/admin_sales/fees.py

```diff
@@ -0,0 +1,52 @@
+from marshmallow_jsonapi import fields
+from marshmallow_jsonapi.flask import Schema
+from flask_rest_jsonapi import ResourceList
+
+from app.api.bootstrap import api
+from app.api.helpers.utilities import dasherize
+from app.models import db
+from app.models.event import Event
+
+
+class AdminSalesFeesSchema(Schema):
+    """
+    Sales fees and revenue for all events
+    """
+
+    class Meta:
+        type_ = 'admin-sales-fees'
+        self_view = 'v1.admin_sales_fees'
+        inflect = dasherize
+
+    id = fields.String()
+    name = fields.String()
+    payment_currency = fields.String()
+    fee = fields.Method('format_fee')
+    revenue = fields.Method('calc_revenue')
+    ticket_count = fields.Method('calc_ticket_count')
+
+    @staticmethod
```

Comment:
Is it require for them to be staticmethod ?

Reply:
It could be a `@property` as well

Reply:
Yes, as they don't use the class itself (`self` would not be used)